components/resources/wallet_strings.grdp

```diff
@@ -125,6 +125,7 @@
   <message name="IDS_BRAVE_WALLET_ROUTE" desc="Swap route label">Route</message>
   <message name="IDS_BRAVE_WALLET_MAX_SLIPPAGE" desc="Swap max slippage label">Max slippage</message>
   <message name="IDS_BRAVE_WALLET_SLIPPAGE_DESCRIPTION" desc="Swap max slippage description">When the deviation between the price of the transaction you submitted and the price at the time of the transaction on chain is greater than this set value, the transaction will fail.</message>
+  <message name="IDS_BRAVE_WALLET_PRICE_IMPACT_DESCRIPTION" desc="Swap price impact description">Price impact refers to the difference between the expected execution price based on market price and trade size.</message>
```

Comment:
This seems a bit confusing. The difference between what and what?

Reply:
@mkarolin I have updated the copy for this! https://github.com/brave/brave-core/compare/4e59ebe958974fc5e99a294dd298fdf05e7f54aa..3faf38adac489c16d52914a31c10aa843c576e9e